src/chunktransformer.ts

```diff
@@ -354,10 +354,10 @@ export class ChunkTransformer extends Transform {
     }) as string;
     this.validateNewRow(chunk, newRowKey);
     if (chunk.familyName && chunk.qualifier) {
-      const row = this.row;
-      row!.key = newRowKey;
-      row!.data = {} as Data;
-      this.family = row!.data![chunk.familyName.value] = {} as Family;
+      const row = this.row!;
+      row.key = newRowKey;
+      row.data = {};
+      this.family = row.data[chunk.familyName.value] = {} as Family;
```

Comment:
am I dreaming, or am I really seeing the good old `a = b = {}` in the real-life JavaScript code? o_O

Reply:
Trying not to change the functionality here, but yeah....

Reply:
You know what, I'm taking this back! I just realized that it might be what they actually wanted to assign _the same_ object to both `this.family` and `row.data[chunk.familyName.value]`. I'm afraid of this code :(